src/scenes/NotFound/NotFound.tsx

```diff
@@ -4,17 +4,21 @@ import Button from 'components/core/Button/Button';
 import Spacer from 'components/core/Spacer/Spacer';
 import Page from 'components/core/Page/Page';
 import GalleryLink from 'components/core/GalleryLink/GalleryLink';
+import { GALLERY_DISCORD, GALLERY_TWITTER } from 'constants/urls';
 
-function NotFound() {
+type Props = {
+  resource?: string;
+};
+
+function NotFound({ resource = 'user' }: Props) {
```

Comment:
updated this page to support not found collections, not just users
![Screen Shot 2022-04-06 at 1 20 50 AM](https://user-images.githubusercontent.com/12162433/161913272-285c5f3c-e70b-4cf2-83b7-ad4e829e1d5a.png)